packages/playground/src/sidebar/runtime.ts

```diff
@@ -165,6 +165,11 @@ function rewireLoggingToElement(
       const prefix = `[<span class="log-${name}">${id}</span>]: `
       const eleContainerLog = eleOverflowLocator()
       allLogs.push(`${prefix}${output}<br>`)
+
+      if (output.includes("Unexpected token 'export'")) {
```

Comment:
I like the idea of reacting to the actual error rather than the emitted JS, but I wonder how fragile this string is. How is output being captured? Could we potentially catch/intercept the actual `SyntaxError` thrown by the `eval`?